src/main/java/Ui.java

```diff
@@ -0,0 +1,51 @@
+public class Ui {
+    private String response;
```

Comment:
I like how you generalise every single command into a single response line then call getResponseDirect() at the end of your UI calls. Very nice!